client/packages/common/src/intl/locales/en/common.json

```diff
@@ -378,6 +380,7 @@
   "error.v6-server-not-configured": "V6 central server not configured",
   "error.v6-server-not-configured-hint": "Check the central server URL",
   "error.vaccine-course-update-failed": "Vaccine course failed to save",
+  "error.value-type-not-correct": "Value type not correct",
```

Comment:
User probably won't understand what this means... since we're only checking if Number is the correct type, maybe we can change the error message to be friendlier